docs/reference/ilm/example-index-lifecycle-policy.asciidoc

```diff
@@ -0,0 +1,182 @@
+[role="xpack"]
+
+[[example-using-index-lifecycle-policy]]
+=== Tutorial: Manage {filebeat} time-based indices
+++++
+<titleabbrev>Manage {filebeat} time-based indices</titleabbrev>
+++++
```

Comment:
Looks like a copy of `docs/management/index-lifecycle-policies/example-index-lifecycle-policy.asciidoc`. Not sure which I'm intended to review....